### PR TITLE
Cleanup octomap dependencies of jsk_pcl_ros

### PR DESCRIPTION
**Pull request by wkentaro**
Following fix in https://github.com/jsk-ros-pkg/jsk_recognition/issues/2087

### CODE REVIEW

File: jsk_pcl_ros/package.xml [hunk 1]
```diff
@@ -110,8 +109,6 @@
   <run_depend>nav_msgs</run_depend>
   <run_depend>octomap</run_depend>
   <run_depend>octomap_server</run_depend>
-  <run_depend>octomap_msgs</run_depend>
-  <run_depend>octomap_ros</run_depend>
```
Comment (wkentaro) [inline jsk_pcl_ros/package.xml]: octomap_ros is not needed and there seems no dependency.

File: jsk_pcl_ros/package.xml [hunk 2]
```diff
@@ -110,8 +109,6 @@
   <run_depend>nav_msgs</run_depend>
   <run_depend>octomap</run_depend>
   <run_depend>octomap_server</run_depend>
-  <run_depend>octomap_msgs</run_depend>
```
Comment (wkentaro) [inline jsk_pcl_ros/package.xml]: `octomap_msgs` has no dynamic link, so it is just a build_depend.

**Comment by wkentaro** [comment]
Sorry for making you confused.
No, this is just a refactoring.

We still need octomap_ros from source if octomap is from-source,
because `jsk_pcl_ros` explicitly depends on `octomap_server`, `octomap_server` depends on `octomap_ros`, in addition, `octomap_ros`, `octomap_server` and `jsk_pcl_ros` depends on `octomap`.

**Comment by mmurooka** [comment]
I now understand. Thank you.

**Comment by wkentaro** [comment]
Could this be merged?

**Pull request merged**